buoyant_cloud/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: Buoyant Cloud
+
+## Overview
+
+[Buoyant Cloud][1] provides fully managed Linkerd on your cluster to monitor the health of Linkerd and deployments. With this integration, you can monitor and be alerted on Linkerd's health, workload traffic, and rollout events.
+
+## Setup
+
+### Installation
+
+You need to have an account at [Buoyant Cloud][1] to use this integration. You can also sign up for Buoyant Cloud in the Datadog Marketplace.
+
+### Configuration
+
+1. Log in to your Buoyant Cloud account.
+2. Browse to the [Buoyant Cloud Settings][2] page, and click `Connect Datadog`.
```

Comment:
Is this step for OAuth? I think it would be easier to direct the user to the Connect Accounts button on the tile, which will be right next to where this text is displayed :)